static/app/components/searchQueryBuilder/formattedQuery.tsx

```diff
@@ -41,7 +41,7 @@ function FilterKey({token}: {token: TokenResult<Token.FILTER>}) {
       <AggregateKeyVisual token={token} />
     </div>
   ) : (
-    <div>{getKeyName(token.key)}</div>
+    <div>{getKeyName(token.key, {showExplicitTagPrefix: true})}</div>
```

Comment:
This was necessary because otherwise `tags[platform]` would just display as `platform` in the recent searches menu.